docs/feature_auto_shift.md

```diff
@@ -283,7 +283,7 @@ shifting one of the keys.
 
 `IGNORE_MOD_TAP_INTERRUPT` changes *only* rolling from a mod tap (releasing it
```

Comment:
This needs to be updated

Reply:
Solved in https://github.com/qmk/qmk_firmware/pull/15741/commits/3a58643f6802a046dc8848572e9ff717769545d1

I find the original paragraphs rather hard to understand so let me know if I misunderstood something in my update.